shimmingtoolbox/cli/b0shim.py

```diff
@@ -325,6 +322,7 @@ def dynamic(fname_fmap, fname_anat, fname_mask_anat, method, opt_criteria, slice
 
                 # If the output format is absolute, add the initial coefs
                 if output_value_format == 'absolute':
+                    initial_coefs = scanner_shim_settings.concatenate_shim_settings(scanner_coil_order)
```

Comment:
intial_coefs is replaced by `options['scanner_shim']` in 
`list_coils = _load_coils(coils, scanner_coil_order, fname_sph_constr, nii_fmap, options['scanner_shim'],
                             json_fm_data['Manufacturer'], json_fm_data['ManufacturersModelName'])`:248
and by `scanner_shim_settings.concatenate_shim_settings(scanner_coil_order)` here. Are they the same thing?

Reply:
They used to be the same thing, now they represent the same thing but are different.

Reply:
the concatenate method makes sure it is the same format as it was before.